pkg/openid4ci/openid4ci.go

```diff
@@ -8,17 +8,20 @@ SPDX-License-Identifier: Apache-2.0
 package openid4ci
 
 import (
+	"bytes"
+	"encoding/json"
 	"errors"
+	"fmt"
+	"io"
+	"net/http"
 	"net/url"
 	"strconv"
-
-	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
 )
 
 // NewInteraction creates a new OpenID4CI Interaction.
 // The methods defined on this object are used to help guide the calling code through the OpenID4CI flow.
 // Calling this function represents taking the first step in the flow.
-// This function takes in an Initiate Issuance Request object from an Issuer (as defined in
+// This function takes in an Initiate Issuance Request object from an IssuerURI (as defined in
```

Comment:
why `issuer` changed to `IssuerURI`?

Reply:
That was an accident due to using the rename function in the IDE - I've fixed it